README.md

```diff
@@ -83,6 +84,29 @@ If you have multiple JFrog platform configurations as secrets, you can use all o
 | Important: When exposing more than one JFrog configuration to the Action, you should always add the ```jf c use``` command to specify the server to use. |
 | --- |
 
+### Setting direct connection details environment variables
+You can use connection details directly by settings one of the following environment variables combinations:
+1. JF_URL (anonymous)
+2. JF_URL + JF_USER + JF_PASSWORD (basic auth)
+3. JF_URL + JF_ACCESS_TOKEN (access token)
+
+You can use them in the workflow as follows:
```

Comment:
You can use them in the workflow as follows:
-->
You can use these environment variable in your workflow as follows: